tasks/scripts/containerd-integration

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e -u -x
```

Comment:
whops, I should get rid of that `-x`

Reply:
https://github.com/concourse/ci/pull/253/commits/5bee7071942eef142bc15c7ae3eb2ef9b02038ec